src/ai/backend/common/redis.py

```diff
@@ -483,3 +489,11 @@ def get_redis_object(
             client=aioredis.Redis.from_url(str(url), **kwargs),
             service_name=None,
         )
+
+
+async def ping_redis_connection(client: aioredis.client.Redis):
+    try:
+        _ = await client.time()
+    except aioredis.exceptions.ConnectionError as e:
+        log.error(str(e))
+        sys.exit(1)
```

Comment:
Please avoid calling `sys.exit()` directly; this will make caller harder to guess which causes program to quit. Instead, just raise relevant `Exception` again. In this case, raising thrown Exception again seems to be sufficient.
```suggestion
        raise e
```